DeusEx/Classes/DXREntranceRando.uc

```diff
@@ -134,94 +162,30 @@ function bool ValidateConnections()
     local MapConnection mapdests[15];
     local int numMaps;
     local int numPasses;
-    local int i,j,k,p;
-    local string curMapName;
+    local int i,j,k;
     local bool foundMap;
-    local int mapidx;
     
     local string canvisit[15];
     local int visitable;
     
     local bool canVisitMap;
-    local bool canVisitDest;
     
     numPasses = numConns;
-    curMapName = "";
-    
-    numMaps = 0;
-    
-    //Get all the map names
```

Comment:
this loop for getting all map names didn't work unless the xfers were sorted by map name, the new GetAllMapNames function fixes that